src/clr-addons/paged-search-result-list/paged-search-result-list.html

```diff
@@ -1,18 +1,23 @@
 <div *ngIf="clrPagerPosition === 'top' || clrPagerPosition === 'both'">
-  <ng-container [ngTemplateOutlet]="pager" [ngTemplateOutletContext]="{ $implicit:'top' }"></ng-container>
+  <ng-container
+    [ngTemplateOutlet]="pager"
+    [ngTemplateOutletContext]="{ $implicit: clrPagerPosition === 'top' || clrPagerPosition === 'both' ? 'top' : undefined }"
```

Comment:
why not just pass 'top'? as this instance here is always top

Reply:
true, as it already has an ngIf in the parent